build/dependencies.props

```diff
@@ -2,9 +2,9 @@
   <PropertyGroup>
     <BenchmarkDotNetPackageVersion>0.11.3</BenchmarkDotNetPackageVersion>
     <GoogleProtobufPackageVersion>3.7.0</GoogleProtobufPackageVersion>
-    <GrpcCorePackageVersion>1.20.0-pre3</GrpcCorePackageVersion>
-    <GrpcCoreApiPackageVersion>1.20.0-pre3</GrpcCoreApiPackageVersion>
-    <GrpcToolsPackageVersion>1.20.0-pre3</GrpcToolsPackageVersion>
+    <GrpcCorePackageVersion>1.21.0-dev201904210946</GrpcCorePackageVersion>
```

Comment:
what exactly do we need from the newer packages?

Reply:
`CallOptions`.